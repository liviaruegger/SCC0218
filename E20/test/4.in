15
agdfabedebfccaedddfgbaegcddcgddgfffgeacfadfgbfcefgggcbadgdcfacdeedgcdbabfbcaefda bbf
bbcdeacdccbdgabegegbbecadgcabbcgbddcfggafegbcdgacgdgbdcgaebff gf
ddecabffacbaeffdcgbacecagdfcdegeddagfedbdcbdafdbbfbgaafbggedebbddfaeefcbbbefadgabecbegfc cd
dggecgcebadgfbaafbfdaffebedfgefgceccfbaccacddadgcaefecaabdbfefcadcfdbabcaaeedbgegd ca
gfgegdbddfbeegdccagcgffcbaadfdfdeabfbgeggagbdfffaecgfddgbbdccgdcede bgdb
dfeedfcfeeafbcfedgcccfbgecdbaefdcbcaadcfddbbecgageecdgddafbaecbagfbgcacabeeaegfffbfcbcgeebbccbb abed
cdgafabccbfeebfagegbefgfdgbcfdbdcbfcecfgaeabdebeecgceffbegccgfc gee
cfdecbeaaedddfbfafbbcccgbffbcgceccgaebagdcgcdaefdcdeagdddfcebecga eggg
egecedgeagacddbcggeaedbcggedfdeegdgdbfbabgbdaedfgccgcebaedcfedcegadfgagacffdaabcgcddafbcge aced
afadcbcbedfcaegebgbefbeabfabdgggcbgccecdadegaegfeda fbe
geacdggdaedfdbeddbcagffadgfebeadfdeccdacdeeabadgfdbcgfefacccagabcbgceafebbfbgbbf ge
dbcbdgdeeaagcgbcagegbbbfdgdgeacdebfdbfddedadaaefcgecdbdbbdcdaegegcdadbbbbfbdbgfgcbgeb cd
gebdcfacffcdbcdbcadeafbgcbegddcaafbefdcfcaggeecfccageeaacdedceegabffafbabffdaecddfdafddedgaea fd
edcababedagbdabadegcbbececgabadabedecbaabfgbagbcdcgagcgggdaaddfdabbgcebeff ggf
ggggeggfdfaedbebcafdecbfdbeebefbeaeaadfdgdagebbbcggddbbge fb
