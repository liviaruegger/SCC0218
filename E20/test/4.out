Not Found

Not Found

1
28 

5
41 49 54 63 72 

Not Found

Not Found

Not Found

Not Found

Not Found

1
21 

1
1 

4
31 52 59 66 

4
37 75 82 85 

Not Found

1
31 

